styles.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  margin: 0;
+}
+
+.headline-container {
+  background-image: url("/images/Header_img.png");
+  background-repeat: no-repeat;
+  background-position: left 100% top 1%;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly correct the position for the background to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/158722834-594fabb1-2fa4-45f3-8d5c-0270cecbf7b3.png) | ![image](https://user-images.githubusercontent.com/35535130/158722995-ae3e93cb-88a6-4e1d-b6f0-26509be139a5.png) |